cloud-recording/reference/rest-api/rest-api-overview.md

```diff
@@ -522,7 +525,7 @@ eventType 43 indicates that the state of the audio stream has changed, and `deta
 - `msgName: `String. The message name, `"web_recorder_reload"`:
 - `reason: `String. The reason for the page reload:
   - `"audio_silence"`: Audio loss issue.
-  -  `"page_load_timeout"`: Web page load timeout. The callback returns this field only under the following conditions: the web page load detection function is enabled, and the web page load timeout. See [Web page load detection](../../develop/webpage-load-timeout).
+  - `"page_load_timeout"`: Web page load timeout. The callback returns this field only under the following conditions: the web page load detection function is enabled, and the web page load timeout. See [Web page load detection](../../develop/webpage-load-timeout).
```

Comment:
1. Please capitalize the first letter after the colon.
2. Could you rephrase 'The callback returns this field only under the following conditions: the web page load detection function is enabled, and the web page load timeout.' so that both parts after the colon are more aligned? I'd say the second one needs a verb (for example, and there is a web page load timeout)